magento-research/pwa-studio
# [doc] usePagination hook docs

## Description

Add JSDoc blocks to the `usePagination.js` source and create a draft topic for the corresponding reference page.

## Related Issue

Closes #1346 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Create the reference docs: `npm run build-reference-doc-snippets`
3. Verify a `usePagination.md` file was created in `src/_includes/auto-generated/peregrine/src/hooks/` with doc block content from the `packages/peregrine/src/hooks/usePagination.js` source file.

## Screenshots / Screen Captures (if appropriate)
n/a

## Checklist:
